contrib/soju.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=soju IRC bouncer service
+Documentation=https://soju.im/
+Documentation=man:soju(1) man:sojuctl(1)
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+User=soju
+Group=soju
+DynamicUser=yes
+StateDirectory=soju
+LogsDirectory=soju
```

Comment:
I don't think we actually need this: soju does not write logs on its own.

Reply:
I can remove that.
I was thinking if it makes sense for an user to configure `log fs /var/log/soju/` ?

then they would be able to do `systemctl clean --what=logs soju.service`

not sure how useful that would be?!

Reply:
This would be pretty confusing, the IRC logs should not be mixed up with application logs...

Reply:
removed that too, the "message store" should probably **not** be in /var/log